Docs - Add draft whitelist sso doc (#1185)
CloudFire-LLC/cloudfire: docs/docs/user-guides/use-cases/nat-gateway/_whitelist-sso.mdx

---

> title: Restrict SSO to VPN Users

Reviewer: ```suggestion
title: Restrict SSO to Firezone Users
```

sidebar_position: 1
description: Configure your identity provider to whitelist traffic from Firezone's WireGuard® based secure access platform.
---

# Restrict application authentication to VPN users

Some identity providers support restricting SSO functionality

> to a pre-defined network zone (IP range). You can set this to the IP of

Reviewer: typo

```suggestion
Some identity providers support restricting SSO access to a predefined CIDR range or list of IP addresses. You can set this to the IP address of
```

your Firezone gateway, so authentication is restricted to users

> connected to the VPN.

Reviewer: ```suggestion
your Firezone gateway to restrict SSO to users connected to Firezone.
```

This adds an additional layer of security to any service that uses
your identity provider for SSO.

We have identified a few common providers that support this
feature and included basic instructions below on how you can set
this up. The pre-requisites to this guide are a deployed Firezone
server with a static IP
([see NAT Gateway](.../nat-gateway))
and ensuring traffic to the identity provider's SSO services are
routed from client devices through the Firezone gateway
([see Split Tunnel](.../split-tunnel)).

## Okta

Okta supports restricting SSO authentication to a "network zone" on
a per application basis.

:::note
If you have split tunneling enabled, ensure the AllowedIPs are
correctly configured to direct traffic to
[Okta's IPs](https://help.okta.com/en-us/Content/Topics/Security/ip-address-allow-listing.htm)
through the Firezone gateway.
:::

:::caution
Sign-on policies are assigned on an application level. We advise
excluding the Firezone application from the IP restriction policy above if Okta
is being used for Firezone authentication.

With
[periodic authentication](../../authenticate#enforce-periodic-re-authentication)

> enabled, you may be locked out Okta entirely when your VPN session

Reviewer: ```suggestion
enabled, you may be locked out of Okta entirely when your VPN session
```
